src/+otp/Problem.m

```diff
@@ -302,7 +296,7 @@ function onSettingsChanged(obj)
             end
         end
         
-        function t = parseTime(obj, t)
+        function t = parseTime(~, t)
```

Comment:
If parseTime does not depend on the object, then it is a static function right? so we can make it static.